arch/riscv/boot/dts/starfive/jh7100-beaglev-starlight-a1.dts

```diff
@@ -20,5 +20,7 @@
 
 &gpio {
 	/* don't reset gpio mux for serial console and reset gpio */
-	starfive,keep-gpiomux = <13 14 63>;
+	starfive,keep-gpiomux = <13 14 63 0 2 3 45>;
 };
+
+#include "codecs/sf_wm8960.dtsi"
```

Comment:
Having random little extra files like this is not how it's done upstream. Please just inline it.